internal/mode/static/nginx/config/servers_template.go

```diff
@@ -1,6 +1,7 @@
 package config
 
 var serversTemplateText = `
+js_preload_object matches from /etc/nginx/conf.d/match.json;
```

Comment:
I'm assuming that if no match pairs exist (we're using a simple HTTPRoute), then this file will still exist, but be empty? Just want to make sure that this doesn't panic in that case where the file may not exist.

Reply:
yes, did verify this. File exists but is empty